app/javascript/tests/enrollment/steps/ExposureInformation.test.js

```diff
@@ -589,27 +533,27 @@ describe('ExposureInformation', () => {
 
   it('Hides "Previous" and "Next" buttons if requisite functions are not passed in via props', () => {
     const wrapper = shallow(<ExposureInformation currentState={{ isolation: false, patient: blankExposureMockPatient, propagatedFields: {} }} patient={blankIsolationMockPatient} showPreviousButton={false} jurisdiction_paths={mockJurisdictionPaths} />);
-    expect(wrapper.find(Button).length).toEqual(0);
+    expect(wrapper.find(Button).length).toEqual(1);
   });
 
   it('Hides the "Previous" button when props.showPreviousButton is false', () => {
     const wrapper = getShallowWrapper(blankIsolationMockPatient);
-    expect(wrapper.find(Button).length).toEqual(1);
-    expect(wrapper.find(Button).at(0).text()).toEqual('Next');
+    expect(wrapper.find(Button).length).toEqual(2);
+    expect(wrapper.find(Button).at(1).text()).toEqual('Next');
   });
 
   it('Clicking the "Previous" button calls props.previous', () => {
     const wrapper = getShallowWrapper(blankIsolationMockPatient, true);
     expect(previousMock).not.toHaveBeenCalled();
-    wrapper.find(Button).at(0).simulate('click');
+    wrapper.find(Button).at(1).simulate('click');
     expect(previousMock).toHaveBeenCalled();
   });
 
   it('Clicking the "Next" button calls validate method', () => {
     const wrapper = getShallowWrapper(blankIsolationMockPatient);
     const validateSpy = jest.spyOn(wrapper.instance(), 'validate');
     expect(validateSpy).not.toHaveBeenCalled();
-    wrapper.find(Button).simulate('click');
+    wrapper.find(Button).at(1).simulate('click');
     expect(validateSpy).toHaveBeenCalled();
   });
```

Comment:
would like to see tests for the new table including button enable/disable

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1125/commits/9a3c80cdc1991ea818b06bb99362ad971f4875d1#